apps/edxapp/templates/services/nginx/dc.yml.j2

```diff
@@ -34,8 +32,12 @@ spec:
                   values:
                   - "edxapp-nginx-{{ deployment_stamp }}"
               topologyKey: kubernetes.io/hostname
+{% if edxapp_image_pull_secret_name is defined and edxapp_image_pull_secret_name is not none %}
```

Comment:
Is it intended to not define it in `vars/all/main.yml` ?

Reply:
yes because there is this `if` clause so it does not need to be defined and the default is public repo?

Reply:
Ok I understand